LookingForSatchels.lua

```diff
@@ -162,10 +162,12 @@ local function initFrame()
         local dungeonID = frame.joinLFG_popupQueue[1]["dungeonID"]
         local v = frame.LFG_dungeonIDs[dungeonID]
         if v and v["status"] and v["status"] == 2 or debugForceShow then --still on watch list, and bonus active
+          local numEncounters, numCompleted = GetLFGDungeonNumEncounters(dungeonID)
+
           frame.joinLFG_lastDungeonID = dungeonID
           frame.joinLFG_lastLfgCategory = frame.joinLFG_popupQueue[1]["lfgCategory"]
           local name = GetLFGDungeonInfo(dungeonID)
-          popupFrame:showDialog("Queue for:\124n"..name)
+          popupFrame:showDialog("Queue for:\124n"..name.."\124n"..numCompleted.." boss(es) done")
```

Comment:
1. Since `GetLFGDungeonNumEncounters` does not always return something we want to show (it returns `0, 0` for LFD dungeons), we should check first if it returned something useful.
2. When we have `numCompleted`, we also get `numEncounters`, so we can show both.

My suggestion is:

```
if numEncounters and numCompleted and numEncounters > 0 then
  popupFrame:showDialog(format("Queue for:\124n%s\124n%s/%s boss(es) done", name, numEncounters, numCompleted))
else
  popupFrame:showDialog(format("Queue for:\124n%s", name))
end
```

Reply:
The issue with `numEncounters` is that it's the total number of encounters for the instance, and not for the specific LFR wing we are queueing, that's the reason why I have avoided using it. I'm not sure what's needed to get just the number of encounters for the wing.

Looking into the `RaidFinder.lua` code at https://www.townlong-yak.com/framexml/live/Blizzard_GroupFinder/RaidFinder.lua#266 it goes into a loop of `GetLFGDungeonNumEncounters` for the dungeon and extracts encounter information with `GetLFGDungeonEncounterInfo`; this may be a little overkill.

Reply:
- `/dump GetLFGDungeonNumEncounters(2649)` shows `3, 0` for me ("The Skittering Battlements", first LFR wing of Nerub-ar Palace)
- `/dump GetLFGDungeonNumEncounters(2651)` shows `2, 2` ("A Queen's Fall", third LFR wing of Nerub-ar Palace)

So these should be the numbers we are looking for. Can you verify this on your end?

Reply:
Yes. Turns out I was wrong and it was just the result of the specific dungeon id I tested it with that showed the whole instance.